server/server.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+//const axios = require('axios');
+//const path = require('path')
+
+const app = express();
+app.use('/', express.static('./public'));
+// app.use(express.static('../dist'))
+
+// app.get('/hello', function(req, res){
```

Comment:
Good job checking to see if your get request works before moving forward with your express server setup.